### DISCUSSION TITLE
Using Armada to compute DAGs

### DISCUSSION THREAD

**Discussion by jaystary** [armadaproject/armada#1415] [General]
Hi, interesting project. I´m curious how could i create and compute a DAG with this (alas take results from one job and feed it downstream to another job).

Would this require integration with a pipelining framework (e.g. ZenML) or can Armada support this out of the box?

Is there somewhere a mid&longer term roadmap, where this projects wants to head?

Thank you.

**Comment by dave-gantenbein** [comment]
Hi Jay-

We're currently working on adding support for ML frameworks to run DAGs within Armada. We're testing an Airflow integration now, and plan to add additional frameworks as time goes on. Eventually we plan to support all the big names, and we'd be happy to collaborate with you on ZenML if that fits your use case.

Regards
-Dave
